natgeo/mortar
# [SYTO-1075] POD Mobile and CTA issues

More follow up tweaks from Kevin regarding changes made for POD Mobile and CTA issues
https://jira.natgeo.com/browse/SYTO-1075

To test:

- Pull SYTO-1075-2
- npm test
- gulp serve

Verify layout across various breakpoints at
http://localhost:3000/components/podpromo/index.html
